index.html

```diff
@@ -312,6 +314,7 @@ <h2>Contact me</h2>
               />
             </li>
             <li>
+              <label id="user-message-label">Message</label>
```

Comment:
- Great job for updating most accessibility, I think you missed updating the color contrast [here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-02-28-at-16-04-37-003?failing) is your result, please update based on this [checker](https://www.aditus.io/button-contrast-checker/) :+1: 
![Screenshot from 2023-02-28 19-32-29](https://user-images.githubusercontent.com/52010789/221918018-dc256e7a-d256-4811-82fe-29ffc55ba733.png)